src/containers/Drawer/index.jsx

```diff
@@ -77,7 +77,7 @@ function ResponsiveDrawer(props) {
   const [selectedIndex, setSelectedIndex] = React.useState(-1);
 
   const handleDrawerToggle = (value) => {
-    if(value === false){
```

Comment:
it is done by prettier ? ( adding spaces don't help...)

Reply:
Logic wise it doesn't but definitely improves readability.